lib/catch2/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Downloaded from:
+#
+# https://github.com/catchorg/Catch2/releases/download/v2.13.9/catch.hpp
+#
+# No library required, because it is just a header.
```

Comment:
For modularity reasons the `set(CATCH2_INCLUDE_DIR)` should be here.
You can also use a CMake target:
```cmake
add_library(catch2 INTERFACE)
target_include_directories(catch2 INTERFACE .)
```

Reply:
Just added this commit:

https://github.com/minetest/minetest/pull/12258/commits/8e60ee62a80fac4d036e0bfe3d6458a76caa97fe